tools/packrom/packrom.cpp

```diff
@@ -304,6 +348,7 @@ int main(int argc,char **argv)
 	}
 
 	memcpy(rom.header.marker,"UZEBOX",MARKER_SIZE);
+	memset(rom.header.reserved, 0, sizeof(rom.header.reserved));
```

Comment:
This line causing cuzebox not to work. I think we should just remove it for now, and just rely on the version number to know what is considered valid data to read from the headers.